### DISCUSSION TITLE
New Awesome secman logo

### DISCUSSION THREAD

**Discussion by abdfnx** [scmn-dev/secman#33] [General]
this is the **New Awesome secman logo**, look at this @iMRxM7mD @Timothee-Cardoso 

# Logo
<img src="https://imgs-secman.web.app/logo.svg" />

# Logo (White)
<img src="https://imgs-secman.web.app/logo_white.svg" />

# Icon
<img src="https://imgs-secman.web.app/icon.svg" align="center" width="150px" />

# Typography
<img src="https://imgs-secman.web.app/typography.svg" align="center" width="150px" />

**Comment by Timothee-Cardoso** [comment]
Wooooooow, This is what I'm talking about, This new logo **expresses security**, The old logo was not expressive, but this is a fingerprint right ?

**Reply by abdfnx** [reply]
yes it is
